instance/models/openedx_appserver.py

```diff
@@ -222,7 +222,7 @@ class OpenEdXAppServer(AppServer, OpenEdXAppConfiguration, AnsibleAppServerMixin
         'configuration_extra_settings',
     ]
 
-    class Meta:
+    class Meta(AppServer.Meta):
```

Comment:
There is magic to prevent that.

This does inherit the `ordering` though, which was the point of this change.